examples/interfaces/IERC1155/IERC1155.rs

```diff
@@ -0,0 +1,71 @@
+// Generated with Sol2Ink v0.2.4
+// https://github.com/Supercolony-net/sol2ink
+
+use brush::traits::AccountId;
+use ink::prelude::string::String;
+use ink::prelude::vec::Vec;
+
+#[ink(event)]
```

Comment:
The problem that we can't generate events outside of the contract, but in future it should be possible, so okay=)